lambda.tf

```diff
@@ -0,0 +1,120 @@
+# Archive with the code to upload
+data "archive_file" "lambda_zip" {
+  type        = "zip"
+  source_dir  = "${path.module}/lambda-src"
+  output_path = "${path.module}/lambda.zip"
+}
+
+# IAM roles and policy
+resource "aws_iam_role" "iam_for_lambda" {
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "lambda.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy" "lambda_policy" {
+  role = "${aws_iam_role.iam_for_lambda.id}"
+  name = "${var.name}-lambda-policy"
+
+  policy = "${data.aws_iam_policy_document.lambda_policy.json}"
+}
+
+data "aws_iam_policy_document" "lambda_policy" {
+  # Allow lambda to log
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+    ]
+
+    resources = [
+      "arn:aws:logs:*:*:*",
```

Comment:
Should we consider limiting this only to LogGroups related to this module? (i.e. Lambda Function)